### DISCUSSION TITLE
Team Leader Permissions (Create Activity)

### DISCUSSION THREAD

**Discussion by Memurame** [kimai/kimai#4141] [Questions & Answers] [answered]
I want to give my team leader the permission to create activities (customers and projects). So far it works as well. However, he does not see any teams in the input field "Team" so that he can assign the activity to a team. There should actually be listed the teams in which the user is himself?

Is this an error or what authorization do I have to assign to him?

Thanks for your help.

![2023-06-28 12_08_05-Kimai – Zeiterfassung](https://github.com/kimai/kimai/assets/15847494/e9fc2365-ecd2-4adb-8fd5-dad13d6daa48)
![2023-06-28 12_08_21-Kimai – Zeiterfassung](https://github.com/kimai/kimai/assets/15847494/f4c4a1ea-bfa4-4999-a944-acf74790c94a)

**Comment by kevinpapst** [comment, marked as answer]
The role Teamlead has nothing to do with being a teamlead of a team.
So your user needs to be one of the teamleads of the team.
